scripts/task_test_blackwell_kernels.sh

```diff
@@ -5,6 +5,7 @@ set -x
 : ${JUNIT_DIR:=$(realpath ./junit)}
 
 pip install -e . -v
+pip install --upgrade "nvidia-cudnn-cu12>=9.11.0.98"
```

Comment:
I think we want to get away from installing packages in the test scripts and instead have them in the Docker container. 
Seems like the docker is set up to install that version: https://github.com/flashinfer-ai/flashinfer/blob/main/docker/install/install_python_packages.sh#L37 ?

What is the version that you were getting?

Reply:
Yes we should avoid adding more installation scripts here.